iam/variables.tf

```diff
@@ -17,3 +17,6 @@ variable "security_audit_group" {
   description = "group giving 'read only' and 'security audit' permissions"
   default     = "security-audit"
 }
+
+variable "environment" {
+}
```

Comment:
Need to add a new line

Reply:
showing new line in my code. Think its github being weird